platform/windows/windows.cmake

```diff
@@ -91,7 +91,7 @@ if(MLN_WITH_EGL)
         mbgl-core
         PRIVATE
             unofficial::angle::libEGL
-            unofficial::angle::libGLESv3
+            unofficial::angle::libGLESv2
```

Comment:
Was this an intended change for `if(MLN_WITH_EGL)`?

Reply:
@tdcosta100 It seems to be causing issues when using the `MLN_WITH_EGL` flag.

`"GLES3/gl2.h": No such file or directory` -> from `gl_functions.cpp`

Reply:
> Was this an intended change for `if(MLN_WITH_EGL)`?

Yes, because there is not a `libGLESv3.dll` even with ANGLE.

> @tdcosta100 It seems to be causing issues when using the `MLN_WITH_EGL` flag.
> 
> `"GLES3/gl3ext.h": No such file or directory` -> from `gl_functions.cpp`

I will check this. Also, apparently the ANGLE port in vcpkg is broken too, so probably I will need to submit a PR there too. As soon as possible I will rework this part.

Reply:
Have you by any chance had any success with this?

Reply:
Hi, @etnav. I did some changes to the ANGLE package and submitted to the vcpkg project. Then things went bad:

https://github.com/microsoft/vcpkg/pull/31254

I can point the vcpkg subdirectory to my branch, so ANGLE works again. But I'm really tired of trying to submit this change to their repository and they are demanding me to do things like I was the package developer. I'm not, I'm just trying to fix something broken in it.

Reply:
Had a glance, that's very annoying. After such a long drag and waste of time. 😠 

Thanks for your effort though, I will try to check this again when I got some spare time and get back to the issue.

Reply:
2 weeks of work, and it wasn't good enough. But ok to stay broken. I will change the ref to my branch, for now.

Reply:
Good news, they finally merged the change, made by someone else they like more. I will change the ref now.

Reply:
Finally fixed!

Reply:
> made by someone else they like more

LOL, they didn't like me either.
I often get into arguments with them, but I get used.
Whatever, thank you again for your efforts!

Reply:
Oh my, looks like my PR. They seem to not like others's contribution, but as they are on GitHub, they are obbligated to.